docs/modules/services/pages/activestorage.adoc

```diff
@@ -0,0 +1,97 @@
+= ActiveStorage
```

Comment:
I was going to mention that I didn't like using Rails nomenclature for these kinds of configurations, but I see that we already have [ActiveJob](https://docs.decidim.org/en/services/activejob.html). So, for consistency, let's leave it as it's and on a future refactor we'll handle both of these pages.